client/src/homepage/latest-news/index.tsx

```diff
@@ -72,21 +76,29 @@ export function LatestNews(props: HydrationData<any>) {
     <section className="latest-news">
       <h2>Latest news</h2>
       <ul className="news-list">
-        {newsItems.map((newsItem) => (
-          <li className="news-item" key={newsItem.url}>
-            <p className="news-title">
-              <span>
-                <NewsItemTitle newsItem={newsItem} />
-              </span>
-              <span>
-                <NewsItemSource newsItem={newsItem} />
+        {newsItems.map((newsItem, index) => {
+          return (
```

Comment:
nit: why the superfluous `{ return (`?

Reply:
Good catch, it's a left-over from when I was passing the same `data-glean` value to both links, before I realized these are different links that should be measured separately. I'll revert.

Reply:
Fixed in deb562522a.